docs/WifiPerformance.md

```
@@ -60,1400 +60,258 @@
-    tx bitrate: 144.4 MBit/s MCS 15 short GI
+    freq: 5500.0
+    RX: 191696 bytes (393 packets)
+    TX: 93135 bytes (469 packets)
+    signal: -36 dBm
+    rx bitrate: 780.0 MBit/s VHT-MCS 9 80MHz VHT-NSS 2
+    tx bitrate: 866.7 MBit/s VHT-MCS 9 80MHz short GI VHT-NSS 2
     ```
 
 #### Realtek 8811AU
 
 <img src=https://stuff.armbian.com/netbox/media/devicetype-images/RTL8812AU.png>
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)